optimum/exporters/onnx/__main__.py

```diff
@@ -444,6 +447,11 @@ def main_export(
             f" `--task {task_non_past} --monolith`, or `--task {task}` without the monolith argument."
         )
 
+    if library_name == "timm" and no_dynamic_axes is False and "vovnet" in model_type:
```

Comment:
```suggestion
    if library_name == "timm" and not no_dynamic_axes and "vovnet" in model_type:
```

Reply:
Added a small test, testing only a few models from transformers and vovnet from timm.